OpenDreamRuntime/DreamMapManager.cs

```diff
@@ -51,13 +51,13 @@ public sealed class Cell {
         };
 
         [Dependency] private readonly IMapManager _mapManager = default!;
-        [Dependency] private readonly IDreamManager _dreamManager = default!;
         [Dependency] private readonly IDreamObjectTree _objectTree = default!;
         [Dependency] private readonly IEntitySystemManager _entitySystemManager = default!;
         private ServerAppearanceSystem _appearanceSystem = default!;  // set in Initialize
 
         public Vector2i Size { get; private set; }
         public int Levels => _levels.Count;
+        public List<DreamObject> AllAtoms { get; } = new();
```

Comment:
Probably worth pre-allocating a large initial capacity.

Reply:
The size of this list after Paradise init was 400252. I'll go ahead and have this pre-allocate 410,000 elements.